bot/__init__.py

```diff
@@ -201,17 +201,27 @@ def clean_argument(arg: str) -> str:
             key = clean_argument(self.buffer[self.index:equals_pos])
 
             # Value: The second argument, specified after the `=`
-            value = (
-                clean_argument(
-                    self.buffer.split("=")[1]
+            rhs = self.buffer.split("=")[1].strip()
```

Comment:
not a fan of this variable name. also you might wanna split it at exactly the first = sign so it doesn't fuck up if there are multiple ='s in the buffer. I mean, consider if buffer was `bot.tags['maths'] = "2+3 =  six"`.

Reply:
renamed to `right_hand`, and splitting will now take the entire **r**ight **h**and **s**ide 🐤